core/src/components/datetime/datetime.tsx

```diff
@@ -988,6 +1019,14 @@ export class Datetime implements ComponentInterface {
     return this.value != null && this.value !== '';
   }
 
+  private isPrevMonthDisabled = () => {
+    return this.workingParts.month <= this.minParts?.month && this.workingParts.year <= this.minParts?.year;
+  }
+
+  private isNextMonthDisabled = () => {
+    return this.workingParts.month >= this.maxParts?.month && this.workingParts.year >= this.maxParts?.year;
+  }
+
```

Comment:
We might want to consider moving this to the `state.ts` utility class. This has other functions that determine whether or not a day/month is disabled.

Additionally, the datetime file is already pretty large, so might be good to reduce the amount of new stuff we put in there.